amr-wind/wind_energy/actuator/wing/wing_ops.H

```diff
@@ -172,7 +190,35 @@ struct ComputeForceOp<
             aflookup(aoa, cl, cd);
 
             // Assume unit chord
-            const auto qval = 0.5 * vmag * vmag * chord[ip] * dx[ip];
+            // If the spanwise eps is zero use the two-dimensional Gaussian
+            // The order of epsilon is (chord, thickness, span)
+            // For the 2D Gaussian the force is normalized on addition with the
+            // wing span in order to conserve the total applied force
+
+            // Calculate factor for qval: 0.5 * Uinf * Uinf * dx
+            auto qfac = 0.5 * vmag * vmag;
```

Comment:
make a ternary expression with the the if condition on line 200.

Reply:
done